src/vm/arm/armsinglestepper.cpp

```diff
@@ -90,7 +90,7 @@ void ITState::Set(T_CONTEXT *pCtx)
 //
 ArmSingleStepper::ArmSingleStepper()
     : m_originalPc(0), m_targetPc(0), m_rgCode(0), m_state(Disabled),
-      m_fEmulatedITInstruction(false), m_fRedirectedPc(false), m_fBypass(false), m_fEmulate(false), m_fSkipIT(false)
+      m_fEmulatedITInstruction(false), m_fRedirectedPc(false), m_fEmulate(false), m_fBypass(false), m_fSkipIT(false)
```

Comment:
Non-Microsoft compilers insist on definition order to match initialization order.